src/stackdriver-nozzle/mocks/heartbeater.go

```diff
@@ -0,0 +1,27 @@
+package mocks
```

Comment:
If we did this we would need to pass in a mock of a logger, and our own ticker and test that the logger got the right thing. I do not like this since now the tests for the users of the heartbeater have to know about the logger. Alternatively, we could we change the interface of the heartbeater to trigger a function instead of logging. This is a better but I do not think it is worth doing at this point.